variable-manipulation.js

```diff
@@ -0,0 +1,23 @@
+const TAX_RATE = 1.1
+const SHIPPING_DEFAULT = 5
+
+function calculateTotal(items, options = {}) {
+  if (items == null || items.length === 0) return 0
```

Comment:
**GPT COMMENT**: 
 - It's recommended to use strict equality (===) instead of loose equality (==) to avoid unexpected type coercion. This can be applied on line 5.